tests/run_examples.py

```diff
@@ -65,7 +70,7 @@
     (jp(examples_rootdir, "sklearnex"), jp(logdir, "sklearnex")),
 ]
 
-availabe_devices = []
+available_devices = []
```

Comment:
When we have an explicit list of available devices, do we even still need the flag `gpu_available`? Wouldn't it be equivalent to saying `"gpu" in available_devices` ?